app/Services/CreateConferenceForm.php

```diff
@@ -0,0 +1,50 @@
+<?php namespace SaveMyProposals\Services;
+
+use Conference;
+use SaveMyProposals\Exceptions\ValidationException;
+use Validator;
+
+class CreateConferenceForm
+{
+    private $rules = [
+        'title' => ['required'],
+        'description' => ['required'],
+        'url' => ['required'],
+        'starts_at' => ['date'],
+        'ends_at' => ['date'],
+        'cfp_starts_at' => ['date'],
+        'cfp_ends_at' => ['date'],
+    ];
+
+    private $input;
+    private $user;
+
+    private function __construct($input, $user)
+    {
+        $this->input = $this->removeEmptyFields($input);
+        $this->user = $user;
+    }
+
+    private function removeEmptyFields($input)
```

Comment:
Does this handle intentionally nulling out a date? Guessing we need to add a test for that? @adamwathan